client/packages/openblocks/src/comps/comps/buttonComp/buttonComp.tsx

```diff
@@ -129,15 +154,23 @@ const ButtonTmpComp = (function () {
             loading={props.loading}
             disabled={
               props.disabled ||
-              (!isDefault(props.type) && getForm(editorState, props.form)?.disableSubmit())
+              (!isDefault(props.type) &&
+                getForm(editorState, props.form)?.disableSubmit())
             }
             onClick={() =>
-              isDefault(props.type) ? props.onEvent("click") : submitForm(editorState, props.form)
+              isDefault(props.type)
+                ? props.onEvent("click")
+                : submitForm(editorState, props.form)
             }
+            style={{
+      margin: `${props.margin.top ? props.margin.top : 0} ${props.margin.right ? props.margin.right : 0} ${props.margin.bottom ? props.margin.bottom : 0} ${props.margin.left ? props.margin.left : 0}`,
```

Comment:
Give seperate marging and padding for all 4 that will be more readable. 

For ex.
margin-top : ${props.margin.top ? props.margin.top : 0}
margin-left : ${props.margin.left ? props.margin.left : 0}